src/pages/verify/e-ids/norwegian-bankid.mdx

```diff
@@ -53,13 +53,14 @@ Two types of Norwegian BankID are available:
 
 Test users are created through the web page at [https://ra-preprod.bankidnorge.no/#/search/endUser](https://ra-preprod.bankidnorge.no/#/search/endUser).
 
-1. Go to the ["TEST NUMBER GENERATOR"](https://ra-preprod.bankidnorge.no/#/generate) to generate a random, valid SSN
-2. It now says "Could not find any bankIDs for ..."
-3. Fill out the first name, last name, and BankID friendly name
-4. Click "Order" to initiate the process
-5. Click the pencil icon and add a phone number and an email that you want to associate with the test user  
+1. Go to the ["TEST NUMBER GENERATOR"](https://ra-preprod.bankidnorge.no/#/generate) to generate a random, valid SSN.  
+_Please ensure that the "Synthetic" checkbox is unchecked before generating a new number. [BankID Biometric](/verify/e-ids/norwegian-bankid/#testing-bankid-biometric) app does not currently support synthetic SSN numbers, so you won't be able to use them with the app._
```

Comment:
synthethic works for non-biometrics login right? So perhaps it should be a conditional statement, "If you want to test biometrics, please make sure to"

Reply:
@mickhansen updated